### ISSUE TITLE
New save confirmation dialogs are not navigable with the keyboard

### ISSUE THREAD

**Issue by GriffinSauce** [microsoft/vscode#116440]
- VS Code Version: 1.53
- OS Version: macOS Big Sur 11.2.1 (20D74)

Steps to Reproduce:

1. Make a change in a file
2. Try to close the tab without saving
3. Confirmation whether you want to save changes will pop up
4. Tab, arrows, space have no effect -> it should be possible to navigate between the buttons like native dialogs.

This breaks the possibility to use VSCode without touching the mouse during these actions. It is especially hurtful when multiple unsaved tabs are closed in succession (CMD+w -> keyboard actions in a loop vs. switching to mouse and back)

Does this issue occur when all extensions are disabled?: Yes

**Comment by jvesouza** [comment]
I tried to reproduce using code insider but found no problems. To close the file that was being edited I used the keystroke ctrl + w.
I don't know if that would make any difference in reproducing the problem.

This problem, if confirmed, also affects people like me who use screen readers to interact with VSCode.

**Comment by GriffinSauce** [comment]
From "ctrl+w" I suspect you're on Windows?

I'm having this problem on macOS, I thought VSCode implemented custom styled dialogs but it appears these are actually the new Big Sur native dialogs. I hope that helps, you're right that it's definitely an accessibility issue as well.

**Comment by jvesouza** [comment]
I forgot to mention that I am on Linux.
Maybe @isidorn can help.

**Comment by isidorn** [comment]
Yeah this is a problem with native dialogs on macOS Big Sur. I see the same and am surprised how keyboard navigation is not working here. Assigning to @deepak1556 for electron
fyi @bpasero

**Comment by bpasero** [comment]
https://brettterpstra.com/2018/06/19/macos-dialog-shortcut-keys/

**Comment by GriffinSauce** [comment]
It appears that a macOS update unchecked that option 🤦 thanks for your patience!

To save others the click:

System Preferences -> Keyboard -> Shortcuts -> check "Use keyboard navigation to move focus between controls" (at the bottom)
